test/test.js

```diff
@@ -1,35 +1,461 @@
-			.end(function (err, res) {
-				expect(err).to.be.null;
-				done();
-			});
+                .post("/wot-test-bench/actions/fastTest")
+                .send({
+                    title: "TestServient",
+                    description:
+                        "Test servient that can be used as a servient to be tested with the WoT Test Bench",
+                    "@context": "https://www.w3.org/2019/wot/td/v1",
+                    "@type": "Thing",
+                    security: ["nosec_sc"],
+                    properties: {
+                        display: {
+                            type: "string",
+                            writable: true,
+                            observable: true,
+                            readOnly: false,
+                            writeOnly: false,
+                            forms: [
+                                {
+                                    href:
+                                        "http://localhost:8081/TestServient/properties/display",
+                                    contentType: "application/json",
+                                    op: ["readproperty", "writeproperty"],
+                                },
+                                {
+                                    href:
+                                        "http://localhost:8081/TestServient/properties/display/observable",
+                                    contentType: "application/json",
+                                    op: ["observeproperty"],
+                                    subprotocol: "longpoll",
+                                },
+                                {
+                                    href:
+                                        "coap://localhost:8082/TestServient/pr/display",
+                                    contentType: "application/json",
+                                    op: ["readproperty", "writeproperty"],
+                                },
+                            ],
+                        },
+                        counter: {
+                            type: "number",
+                            writable: true,
+                            observable: true,
+                            readOnly: false,
+                            writeOnly: false,
+                            forms: [
+                                {
+                                    href:
```

Comment:
would be better to not split lines here?

Reply:
Was done by prettier automatically.